packages/checkout/core/src/components/plug-checkout/plug-checkout.service.ts

```diff
@@ -96,6 +99,56 @@ export class PlugCheckoutService {
     }
   }
 
+  public async getPaymentSession() {
+    if (!settings.paymentSessionKey) {
+      return
+    }
+
+    await new Promise(f => setTimeout(f, 3000));
```

Comment:
Deus ta vendo essa gambeta para simular um loading 👀

Reply:
HAHAHAHAHAHAHA

it's necessary